daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -1017,7 +1017,7 @@ private[lf] final class Compiler(
       ifaceId: TypeConName,
       choice: InterfaceChoice,
   ): (SDefinitionRef, SDefinition) =
-    topLevelFunction(ChoiceDefRef(ifaceId, choice.name), 2) { case List(cidPos, choiceArgPos, _) =>
+    topLevelFunction(ChoiceDefRef(ifaceId, choice.name), 2) { case List(cidPos, choiceArgPos) =>
```

Comment:
I think the actual fix should be this, based on the other ChoiceDefRef (for template choice exercises):

```suggestion
    topLevelFunction(ChoiceDefRef(ifaceId, choice.name), 3) { case List(cidPos, choiceArgPos, _) =>
```

Reply:
cc @remyhaemmerle-da

Reply:
haha, I just did that, but @remyhaemmerle-da thinks it's the other one.

Reply:
(the third argument being a `tokenPos` isn't used)